sirepo/html.py

```diff
@@ -70,11 +70,13 @@ def _widget_supported_codes(ctx):
               </button>
               <ul class="dropdown-menu" aria-labelledby="sr-landing-supported-codes">
     '''
-    x = PKDict([(s, s) for s in sirepo.feature_config.cfg().sim_types])
+    x = PKDict([(s, s) for s in sirepo.feature_config.PROD_FOSS_CODES])
```

Comment:
I think this issue still needs to be open, because this should be an intersection at least.

Reply:
I opened a new ticket as #3632